src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
+// three lines.
+/* jshint ignore: start */
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
I think he means the Compat branch or jquery-compat package which is I guess what we'll call it up on npm. It's a full jQuery with all the IE8 crap in it tho, not a shim. I guess you could consider jquery-migrate to be a shim but I don't think it would have a role here.